octave/ldpcut.m

```diff
@@ -214,9 +214,20 @@
   mod_order = 16; bps = log2(mod_order);
   modulation = 'QAM'; mapping = ""; demod_type = 0; decoder_type = 0;
   max_iterations = 100; EsNo_dec = 10;
-
+  qam16 = [
+    1 + j,  1 + j*3,  3 + j,  3 + j*3;
+    1 - j,  1 - j*3,  3 - j,  3 - j*3;
+   -1 + j, -1 + j*3, -3 + j, -3 + j*3;
+   -1 - j, -1 - j*3, -3 - j, -3 - j*3];
+  qam16 = qam16/std(qam16(:));
```

Comment:
std divides the sum of squares by N-1; so I think v=qam16(:);  qam16 = qam16/ (sqrt(v'*v/16)) should be slightly more accurate